### ISSUE TITLE
Feedback: Standard Layers

### ISSUE THREAD

**Issue by cabergh** [whitead/dmol-book#87]
- In 7.1: I think it would be nice with some figure showing the layout of an example neural network, e.g. showing some typical hyperparameters or layer designs. In the two first chapters about deep learning, there’s no sketch of a typical neural network.

- In 7.2.1 Convolutions: “...the trainable parameters doesn’t...” -> “...parameters don’t...”

- In 7.2.1 Convolutions: “This can either be treated by padding…”. Maybe you can explain a bit more what padding is? E.g. what numbers/elements are added to the arrays?

- In 7.2.1 Convolutions: “...will be one-hot indicators for the base (T, C, A, G).” Can you explain what a one-hot indicator is? E.g. as side note?

- Side note in 7.2.1: “It’s a bit more complicated.” I don’t think you should start the note like this. Compared to what? It’s not completely clear when this side note should be read, so I believe the reference has to be repeated. I also think this side note should explain what equivariance is before going into detail.

- In 7.2.2 Pooling: “Read more about pooling here” . is missing.

- In 7.4 Example: “We need to then make sure our final layer is a sigmoid.” Why does it have to be sigmoid? This is not explained.

- In 7.4 Example: “2 dimensional” -> “2-dimensional” or “two-dimensional”

- In 7.4 Example: “Our kernel filter size for the 1D convolution will be 5 and we’ll use 16 filters.” Why were these hyperparameters chosen? I think this should be explained even if it’s perhaps a random choice.

- In 7.4 Example: “However, we still able to learn a little bit...” -> “However, we are still able to learn some things…”.

- In 7.5 Regularization: “Thus we view regularization…” -> “Thus, we view...”

- In 7.5.1 Early Stopping: “...move a fix distance...”  -> “...move a fixed distance…”

- In 7.5.2 Weight: “...on the trainable weights in the model like we saw before.” -> “...in the model we saw before.” Also, it would be nice if you could remind the reader which model this is!

- In 7.5.4 Batch Normalization: “This is what batch normalization.” This sentence is not complete. “This is what batch normalization accomplishes”?

- In 7.5.4.1 Layer Normalization: “...behave well centered…” -> “...well-centered…”

- In 7.5.4.1 Layer Normalization: “...with both take variable sized inputs.” -> “...both taking variable-sized inputs.”

- In 7.5.5 Dropout: I would reformulate “Despite it appearing ad-hoc, it is effective.” to “Despite the ad-hoc appearance of the dropout layer, it’s effective.”

- In 7.5.5 Dropout: “You need to choose the dropout rate when using it, another hyperparameter.” -> “...using it, which means an additional hyperparameter”.

- In 7.6 Residues: ‘“adding” or “concatonating”’ -> “concatenating”?

- In 7.6. Residues: “a gradient can propogate” spelling mistake -> “propagate”

- In 7.6 Residues: “You can also be more complex and use a trainable function...”. I don’t think the point is for the person to be more complex. Change to e.g. “You can also aim for more complexity and use…”

- In 7.6 Residues: “...is one of the least effective thing to adjust.” -> “...things to adjust”

- In 7.6 Residues: “So don’t expect much of an improvement.” I don’t think this should be a separate sentence.

**Comment by whitead** [comment]
Fixed by ce30de8
